corehq/apps/reports/static/reports/js/case_details.js

```diff
@@ -288,5 +288,37 @@ hqDefine("reports/js/case_details", [
 
     });
 
+    function toggleAccordionArrowIcon(panelHeading) {
+      var accordionArrowIcon = panelHeading.find('.accordion-arrow-icon');
+      accordionArrowIcon.toggleClass('fa-angle-double-down fa-angle-double-right');
+    }
+
+    $(function() {
+      var allPanels = $('#case-properties-accordion').find('.panel-collapse');
+      var allPanelHeadings = $('#case-properties-accordion').find('.panel-heading');
+
+      // Add click event listener to Expand All button
+      $('#expand-all-accordion-btn').click(function() {
+        allPanels.collapse('show');
+        allPanelHeadings.each(function() {
+          toggleAccordionArrowIcon($(this));
```

Comment:
If some panels are closed and some are open, then the user clicks "Expand All", will this make all arrows :arrow_down_small: or will it flip all, so those that were already :arrow_down_small: become :arrow_forward: ?

Reply:
It will make all 
![image](https://github.com/dimagi/commcare-hq/assets/14318173/46c808c8-72cd-47ca-97ea-cac23fd96e7a)

https://github.com/dimagi/commcare-hq/assets/14318173/14b227d2-1dab-4569-8af3-5d6d8db467e7